python3/extensions/pool_update.precheck

```diff
@@ -250,7 +253,7 @@ if __name__ == '__main__':
             update_vdi_uuid = session.xenapi.VDI.get_uuid(update_vdi)
             update_vdi_valid = True
         except Exception as e:
-            print(failure_message(CANNOT_FIND_UPDATE, []))
+            print((failure_message(CANNOT_FIND_UPDATE, [])))
```

Comment:
May I know why () is added for all the print, thank you~

Reply:
If you want to group the arguments for readability or to make it clear that you're calling a function with two arguments, you can keep the outer parentheses when we black to format code this done as part of .Black is a PEP 8 compliant opinionated formatter.

Reply:
Thanks, do you mean it is done by this black formatter: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html ?
But I don't see this format there.
BTW, if we choose some formatter, we should add it in the repo to keep all the python code follow the same format?

Reply:
Ack about using same format on repo  , reverted to original format.